src/main_psp/1A794.c

```diff
@@ -28,6 +28,14 @@ typedef struct {
     s32 unk8;
 } Unk08919D98;
 
+typedef struct {
+    s32 unk0;
+    s32 unk4;
+} Unk08C4218C;
```

Comment:
Point32? Or unlikely?

Reply:
It's more some kind of internal struct. I think it looks identical to `Point32`. But it would be a better idea to keep structs separate from `main` and have it completely isolated from the actual game :thinking: